config.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+type Project struct {
+	Organization string
+	Name         string
+	Version      string
+}
+type Config struct {
+	Project *Project
+}
+
+var config *Config
+
+func main_config(cmdRoot *cobra.Command) {
+	GenerateConfig()
+
+	var cmdConfig = &cobra.Command{
+		Use:   "config",
+		Short: "Configuration for wrench",
+		Long:  `configuration picked up by wrench and used in commands`,
+		Run: func(cmd *cobra.Command, args []string) {
+			d, err := yaml.Marshal(&config)
+			if err != nil {
+				panic(err)
+			}
+			fmt.Printf(string(d))
+		},
+	}
+
+	cmdRoot.AddCommand(cmdConfig)
+}
+
+func GenerateConfig() {
+	var project = &Project{
+		Organization: detectProjectOrganization(),
+		Name:         detectProjectName(),
+		Version:      detectProjectVersion(),
+	}
+
+	config = &Config{
+		Project: project}
+}
+
+func detectProjectOrganization() string {
+	out, err := exec.Command("sh", "-c", "hostname -f").Output()
```

Comment:
The username you want to push docker images to is not always in hostname. For example if the building is done in a bunch of expendable Cloud VMs. Maybe it is simpler to make it configurable rather than try to parse it from FQDN?